go.mod

```diff
@@ -219,7 +219,7 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.13.5 => github.com/ethereum-optimism/op-geth v1.101305.2-rc.2
+replace github.com/ethereum/go-ethereum v1.13.5 => github.com/ethereum-optimism/op-geth v1.101305.2-rc.2.0.20240116235610-75ee01e84295
```

Comment:
We'll need to point this to the commit that winds up on the optimism branch (hopefully it's a fast forward merge and so is the same but will verify once it has).

Reply:
updated